Open Judge System/Services/OJS.Services/Business/Participants/ParticipantsBusinessService.cs

```diff
@@ -146,6 +152,26 @@ public void UpdateTotalScoreSnapshotOfParticipants()
             this.participantsData.UpdateTotalScoreSnapshot();
         }
 
+        public void RemoveParticipantMultipleScores()
+        {
+            var participants = 
+                this.scoresDataService.GetAll()
+                .GroupBy(ps => new { ps.IsOfficial, ps.ProblemId, ps.ParticipantId })
+                .Where(ps=>ps.Count() > 1)
+                .ToList();
```

Comment:
This is `participantScores` not `participants`

Reply:
Correct, we have multiple participantScores not multiple participants.